### PR TITLE
Adjusts keepalive and timeout settings for gunicorn

### PR DESCRIPTION
**Pull request by limed**
This adjusts the timeout keepalive settings for  gunicorn, also managed to sneak in a connection draining tweak for the ELB. See more details on connection draining here:

https://aws.amazon.com/blogs/aws/elb-connection-draining-remove-instances-from-service-with-care/

This should be done after we change the gunicorn worker class to `gevent`

### CODE REVIEW

**Comment by limed** [comment]
A by product of this change requires us to run the `make k8s-svc` command which changes the ELB idle timeout setting from 118 seconds to 60 seconds (which is its default). Note that running this command will not redeploy the ELB so its safe to run.

We will also need a redeployment of services to adjust the new gunicorn settings. The command `make k8s-deployments` should solve this.

**Comment by limed** [comment]
We should not be merging this until #430 is merged and tested

File: apps/mdn/mdn-aws/k8s/regions/oregon/stage.mm.sh [hunk 1]
```diff
@@ -53,7 +53,8 @@ export WEB_NAME=web
 export WEB_REPLICAS=1
```
Comment (escattone) [inline apps/mdn/mdn-aws/k8s/regions/oregon/stage.mm.sh:53]: We're missing:
```sh
 export ELB_CONNECTION_DRAINING_ENABLED=True
```
in this file.

Comment (limed) [reply apps/mdn/mdn-aws/k8s/regions/oregon/stage.mm.sh:55]: Good catch, fixed in ad410a1b368806701ade90f364d2c4fe859e2cd4

**Pull request merged**